mail_footer_notified_partner/models/mail_followers.py

```diff
@@ -0,0 +1,45 @@
+# © 2016 ACSONE SA/NV <https://acsone.eu>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api
+from odoo.tools.translate import _
+
+
+class MailNotification(models.Model):
+    _inherit = 'res.partner'
+
+    @api.multi
+    def _notify_by_email(
```

Comment:
this method is not any-more available in V 11.0
https://github.com/odoo/odoo/blob/10.0/addons/mail/models/res_partner.py#L171
not in https://github.com/odoo/odoo/blob/11.0/addons/mail/models/res_partner.py#L114